src/components/career-form/career-components.tsx

```diff
@@ -17,6 +17,7 @@ import { rgba } from 'polished';
 import { RightArrowIcon } from '../icons/buttons-icons/right-arrow';
 import { FieldErrors } from 'react-hook-form';
 import { Headline } from './career-form';
+import { Trans, useTranslation } from 'react-i18next';
```

Comment:
Sometimes you import those functions from `react-i18next` and sometimes from `gatsby-plugin-react-i18next`. Is there a reason behind this?